### ISSUE TITLE
Axis Window Shade

### ISSUE THREAD

**Issue by jbloomfield3** [iklein99/homebridge-smartthings#87]
**Describe The Bug:**

**To Reproduce:**

**Expected behavior:**

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```
11/18/2022, 8:49:50 PMSmartThings Ik BridgeBridge is running on port 51836.
11/18/2022, 8:49:51 PMSmartThings Ik BridgeSmartthings Plug (IK)Restoring existing accessory from cache: living room shade
11/18/2022, 8:49:51 PMSmartThings Ik BridgeSmartthings Plug (IK)Restoring existing accessory from cache: dining room shade
11/18/2022, 8:49:51 PMSmartThings Ik BridgeSmartthings Plug (IK)Restoring existing accessory from cache: Light Switch
11/18/2022, 8:49:52 PMSmartThings Ik BridgeSmartthings Plug (IK)WARNINGliving room shade battery level LOW
11/18/2022, 8:49:52 PMSmartThings Ik BridgeSmartthings Plug (IK)WARNINGdining room shade battery level LOW
11/18/2022, 8:50:04 PMSmartThings Ik BridgeSmartthings Plug (IK)WARNINGliving room shade battery level LOW
11/18/2022, 8:50:04 PMSmartThings Ik BridgeSmartthings Plug (IK)WARNINGdining room shade battery level LOW
11/18/2022, 9:15:54 PMSmartThings Ik BridgeSmartthings Plug (IK)WARNINGliving room shade battery level LOW
11/18/2022, 9:15:54 PMSmartThings Ik BridgeSmartthings Plug (IK)WARNINGdining room shade battery level LOW

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Plugin Version**:
* **Homebridge Version**: 
* **Node.js Version**: 
* **NPM Version**: 
* **Operating System**:

**Comment by jbloomfield3** [comment]
The app was easy to install it just wont operate my Axis Gear Window operator.

**Comment by iklein99** [comment]
Fixed in version 1.4.19
